recipes/gtk/all/conanfile.py

```
@@ -57,13 +59,19 @@ def _gtk3(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+            # Fix duplicate definitions of DllMain
+            self.options["gdk-pixbuf"].shared = True
+            # Fix segmentation fault
+            self.options["cairo"].shared = True
```

Please also raise an exception in validate(self) if these two options are False